WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/ShippingLabelPackagesFormViewModel.swift

```diff
@@ -13,6 +13,14 @@ final class ShippingLabelPackagesFormViewModel: ObservableObject {
         selectedPackages.count > 1
     }
 
+    /// Message displayed on the Move Item action sheet.
+    ///
+    @Published private(set) var moveItemActionSheetMessage: String = ""
```

Comment:
Q: Why not using an optional string here? Also if it needs to be unwrapped, it's more clear in my opinion.

Reply:
Since the `Text` view for the title of the action sheet require a non-optional string, I think it's best that I use the view model as the single source of truth instead of using nil coalescing in the view. That makes sense right?